src/config.rs

```diff
@@ -58,6 +53,7 @@ impl ConfigurationBuilder {
                 .unwrap_or_else(|| 1000.to_string())
                 .parse()
                 .expect("Block build step should be a number."),
+            mode: self.mode.unwrap_or_else(|| "strict".to_string()),
```

Comment:
I think it's better to use `unwrap_or("strict".to_string())`, we don't need a lambda function here.

Reply:
@eadventurous - clippy thinks that it worth it)
But if we will place variables in constants we will please static analysis gods.

Reply:
Unfortunately no - we can't place `String` in const and we can't use `&'static str` for dynamic sized values(